api/action/popups/b.html

```diff
@@ -5,8 +5,28 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <style>
+    .center {
```

Comment:
Should you unify this CSS in an external file? It's the same for many popups.

Reply:
Yep, I was just being lazy :)